plugins/jvm-abi-gen/src/org/jetbrains/kotlin/jvm/abi/JvmAbiClassBuilderInterceptor.kt

```diff
@@ -28,9 +28,11 @@ enum class AbiMethodInfo {
     STRIP,
 }
 
+data class Member(val name: String, val desc: String)
+
 sealed class AbiClassInfo {
     object Public : AbiClassInfo()
-    class Stripped(val methodInfo: Map<Method, AbiMethodInfo>, val prune: Boolean = false) : AbiClassInfo()
+    class Stripped(val methodInfo: Map<Method, AbiMethodInfo>, val fieldsInAbi: Set<Member>, val prune: Boolean = false) : AbiClassInfo()
```

Comment:
I'd prefer to unify `methodInfo` and `fieldsInAbi` in `memberInfo: Map<Member, AbiMethodInfo>` here but tried to make as few changes as possible. 

Please let me know if you ok with that and I will refactor it to use a single collection instead.